pygments/lexers/hlb.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+"""
+    pygments.lexers.hlb
+    ~~~~~~~~~~~~~~~~~~
+
+    Lexers for HLB.
+
+    :copyright: Copyright 2006-2019 by the Pygments team, see AUTHORS.
+    :license: BSD, see LICENSE for details.
+"""
+
+from pygments.lexer import RegexLexer, bygroups
+from pygments.token import *
+
+import re
+
+__all__=['HlbLexer']
+
+class HlbLexer(RegexLexer):
+    """
+    Lexer for `HLB <https://github.com/openllb/hlb>`_ source code.
+
+    .. versionadded:: 2.6
+    """
+    name = 'Hlb'
+    aliases = ['hlb']
+    filenames = ['*.hlb']
+    flags = re.MULTILINE | re.UNICODE
+
+    tokens = {
+        'root' : [
+            (u'(#.*)', bygroups(Comment.Single)),
+            (u'((\\b(0(b|B|o|O|x|X)[a-fA-F0-9]+)\\b)|(\\b(0|[1-9][0-9]*)\\b))', bygroups(Number)),
```

Comment:
`(b|B|o|O|x|X)` -> `[bBoOxX]`

Also, you don't need `bygroups` for a single token here and everywhere, just put the token type directly. That also allows you to remove the groups enclosing everything in all regexes.